client/my-sites/add-ons/main.tsx

```diff
@@ -56,6 +57,40 @@ const ContainerMain = styled.div`
 	}
 `;
 
+export const StorageAddonsNotice = () => {
+	const translate = useTranslate();
+	const selectedSite = useSelector( getSelectedSite ) ?? null;
+	const availableStorageAddOns = AddOns.useAvailableStorageAddOns( { siteId: selectedSite?.ID } );
+	const storageAddOnPurchases = Purchases.useSitePurchasesByProductSlug( {
```

Comment:
FYI I've just deployed the PR introducing `usePurchasedStorageAddOn()` here: https://github.com/Automattic/wp-calypso/pull/97862/files#diff-cb4c57e0d7623ae5161e3b4336ff3236e9143ca3494f0b5492a0592e6ba4773fR1

To clarify, it's not necessary to apply it, since it does a round of mapping the product slug to the add-on slug that's redundant here. The usage here makes me rethink that it'd be great if we can deprecate the idea of add-on slugs which is calypso-only. Maybe just use the pair of product slug and quantity as the add-on identifier would be better 🤔